sn_client/src/wallet.rs

```diff
@@ -343,15 +343,15 @@ impl WalletClient {
     /// Send tokens to nodes closest to the data we want to make storage payment for. Runs mandatory verification.
     ///
     /// # Arguments
-    /// - content_addrs - [Iterator](Iterator)<Items = [NetworkAddress](NetworkAddress)>
```

Comment:
`content_addrs` refers to an iter of `NetworkAddress` right?

Reply:
Yes, correct. Can I improve this? Let me know! 👍

Reply:
I think you might have changed that to `NanoTokens` here.

Reply:
oh yes! I think I made a mistake with the find and replace feature. Thank you. Well spotted!!